FHIR-Converter/client/src/core/converter/converter-factory.ts

```diff
@@ -34,36 +33,26 @@ export class ConverterEngineFactory {
 			throw new ConversionError(localize('message.resultFolderNotExits', resultFolder));
 		}
 
-		let engine;
-		const converterType = globals.settingManager.getWorkspaceConfiguration(configurationConstants.ConverterTypeKey);
-		if (!converterType) {
-			throw new ConfigurationError(localize('message.noConverterTypeProvided'));
+		// Check that the template folder is available
+		const templateFolder: string = globals.settingManager.getWorkspaceConfiguration(configurationConstants.TemplateFolderKey);
+		if (!templateFolder) {
+			throw new ConfigurationError(localize('message.noTemplateFolderProvided'));
 		}
 
-		if (converterType === ConverterType.hl7v2ToFhir) {
-			// Check that the template folder is available
-			const templateFolder: string = globals.settingManager.getWorkspaceConfiguration(configurationConstants.TemplateFolderKey);
-			if (!templateFolder) {
-				throw new ConfigurationError(localize('message.noTemplateFolderProvided'));
-			}
-
-			// Check if template folder exists
-			if (!fs.existsSync(templateFolder)) {
-				throw new ConversionError(localize('message.templateFolderNotExits', templateFolder));
-			}
-
-			// Check that the entry template is available
-			const rootTemplate = globals.settingManager.getWorkspaceState(stateConstants.TemplateKey);
-			if (!rootTemplate) {
-				throw new ConversionError(localize('message.needSelectTemplate'));
-			}
+		// Check if template folder exists
+		if (!fs.existsSync(templateFolder)) {
+			throw new ConversionError(localize('message.templateFolderNotExits', templateFolder));
+		}
 
-			// create the engine
-			engine = new Hl7v2FhirConverterEngine(templateFolder, rootTemplate, resultFolder);
-		} else {
-			throw new ConversionError(localize('message.converterEngineNotSupported', converterType));
+		// Check that the entry template is available
```

Comment:
We can use `root template` here instead of `entry template`, because in C-CDA there is a type of snippet called `entry`.

Reply:
Changed the string in this comment.